ipa-core/src/config.rs

```diff
@@ -275,6 +276,10 @@ pub struct ServerConfig {
 
     /// Configuration needed for encrypting and decrypting match keys
     pub hpke_config: Option<HpkeServerConfig>,
+
+    // Sets the SETTINGS_MAX_CONCURRENT_STREAMS option for HTTP2 connections.
+    // [`hyper_util::server::conn::auto::Http2Builder::max_concurrent_streams`]
+    pub max_concurrent_stream: Option<u32>,
```

Comment:
this should go to `Http2Configurator`

Reply:
I have been investigating how this works in hyper and it seems like SETTINGS_MAX_CONCURRENT_STREAMS is a server only parameter. I was reading the RFC and it also says only the peers (the endpoints) can set this value. Hence I think it makes sense to keep this parameter where it's, in the ServerConfig.

Reply:
Ok, I am convinced that it should be the server-side setting, but `ServerConfig` is not the right place for it. I just merged https://github.com/private-attribution/ipa/pull/1104 that creates server-side config, this should go there I think